### PR TITLE
Make plugit usable as library

### PR DESCRIPTION
**Pull request by the-glu**
Allows plugit to be used as a library for mail functions + sqlachemy

### CODE REVIEW

File: plugit/__init__.py [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
 from params import PI_BASE_URL
 
-app = Flask("sample-project", static_folder='media', static_url_path=PI_BASE_URL+'media')
-# For SqlAlchemy
-#app.config['SQLALCHEMY_DATABASE_URI'] = config.SQLALCHEMY_URL
-#db = SQLAlchemy(app)
 
-def load_actions(act_mod):
-    routes.load_routes(app, act_mod)
+app = Flask(getattr(config, 'PI_PROJECT_NAME', 'PlugIT-Project'), static_folder='media', static_url_path=PI_BASE_URL+'media')
```
Comment (brukau) [inline plugit/__init__.py]: '+' used to concatenate strings

**Comment by brukau** [comment]
I didn't understand which change makes it possible to use plugit as a library

**Comment by the-glu** [comment]
Before the mail callback has to be coded in the main entry point for flask (`server.py`). Now the function can be anywhere, allowing a generic `server.py`

File: plugit/__init__.py (hunk 1 above)
Comment (the-glu) [inline plugit/__init__.py]: Fixed

File: plugit/__init__.py [hunk 2]
```diff
@@ -1,17 +1,13 @@
...
-# For SqlAlchemy
-#app.config['SQLALCHEMY_DATABASE_URI'] = config.SQLALCHEMY_URL
-#db = SQLAlchemy(app)
 
-def load_actions(act_mod):
-    routes.load_routes(app, act_mod)
+app = Flask(getattr(config, 'PI_PROJECT_NAME', 'PlugIT-Project'), static_folder='media', static_url_path='{}media'.format(PI_BASE_URL))
+
+
+def load_actions(act_mod, mail_callback=None):
```
Comment (brukau) [inline plugit/__init__.py]: could you add docstring here to explain what is the purpose of mail_callback

File: plugit/__init__.py [hunk 3]
```diff
@@ -1,17 +1,13 @@
...
 from params import PI_BASE_URL
 
-app = Flask("sample-project", static_folder='media', static_url_path=PI_BASE_URL+'media')
-# For SqlAlchemy
-#app.config['SQLALCHEMY_DATABASE_URI'] = config.SQLALCHEMY_URL
-#db = SQLAlchemy(app)
 
-def load_actions(act_mod):
-    routes.load_routes(app, act_mod)
+app = Flask(getattr(config, 'PI_PROJECT_NAME', 'PlugIT-Project'), static_folder='media', static_url_path='{}media'.format(PI_BASE_URL))
```
Comment (brukau) [inline plugit/__init__.py]: would be good to split the line in two

**Comment by the-glu** [comment]
Both comments fixed :)

**Pull request merged**